src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,69 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { addComment } from '../../api/api';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+export const NewCommentForm = ({ postId, loadComments }) => {
+  const [name, changeName] = useState('');
+  const [email, changeEmail] = useState('');
+  const [body, changeBody] = useState('');
 
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={(event) => {
+        event.preventDefault();
+        addComment(name, email, body, postId)
+          .then(loadComments);
+        changeName('');
+        changeEmail('');
+        changeBody('');
+      }}
+    >
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          value={name}
+          onChange={event => changeName(event.target.value)}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create method setCommentPart which should work like this:
```javascript
const setCommentPart = (key, value) => {
  setComment(prevComment => ({
    ...prevComment,
    [key]: value,  
  })
}
```

Reply:
done